apps/sms/views/conversation/js/recipients.js

```diff
@@ -716,7 +717,6 @@
       }
 
       state.isTransitioning = false;
-      this.removeEventListener('transitionend', te, false);
```

Comment:
but this is the only change in this file here, maybe I should not change this now?